pkg/operator/ceph/cluster/controller.go

```diff
@@ -326,6 +328,16 @@ func (c *ClusterController) onAdd(clusterObj *cephv1.CephCluster, ref *metav1.Ow
 		return nil
 	}
 
+	if clusterObj.Spec.Network.IPFamily == cephv1.IPv6 {
+		k8sVersion, err := k8sutil.GetK8SVersion(c.context.Clientset)
+		if err != nil {
+			return errors.Wrapf(err, "error finding Kubernetes version")
+		}
+		if !k8sVersion.AtLeast(version.MustParseSemantic("v1.13.0")) {
+			return errors.New(fmt.Sprintf("failed to create cluster on IPv6. Minimum Kubernetes version required: 1.13.0. Actual version: %s", k8sVersion.String()))
```

Comment:
Use `errors.Errorf` instead of ` errors.New(fmt.Sprintf())`

Reply:
fixed.